docker-sync.yml

```diff
@@ -0,0 +1,11 @@
+version: "2"
```

Comment:
Docker-sync is not required anymore as Docker on MacOS has improved performances just by using the `:cached` option on mounted volumes.
Meaning: this file is not needed.